src/libs/ReportUtils.ts

```diff
@@ -2697,7 +2698,7 @@ function hasReportNameError(report: OnyxEntry<Report>): boolean {
  */
 function getParsedComment(text: string): string {
     const parser = new ExpensiMark();
-    return text.length <= CONST.MAX_MARKUP_LENGTH ? parser.replace(text) : lodashEscape(text);
+    return text.length <= CONST.MAX_MARKUP_LENGTH ? parser.replace(text, {shouldEscapeText: !shouldAllowRawHTMLMessages()}) : lodashEscape(text);
```

Comment:
`shouldEscapeText` is `true` by default: https://github.com/Expensify/expensify-common/blob/45d3b61bb38b4f9a19ddf573ce1e212369b242db/lib/ExpensiMark.js#L480